### ISSUE TITLE
Apple M1

### ISSUE THREAD

**Issue by elextr** [geany/geany-osx#23]
It has been reported on the ML that the standard bundle works on Apple M1 silicon, @techee do you want to add a note in README to the effect that its reported to work but not tested by you (unless you do test on M1 :)

**Comment by techee** [comment]
I actually have the Mac Mini developer transition kit prototype from Apple which doesn't use M1 but some of their previous ARM chips used on iPad. And I tested Geany something like 9 month ago so it is an old news it works :-).

I also tried compiling the whole app bundle natively for ARM but some of the dependencies in gtk-osx failed to build but I expect it will change once the upstream projects get updated. I don't expect that it will ever be possible to make an universal build (using fat binaries with both ARM and x86 code) because every single dependency would have to be updated to support that and I don't think it's realistic. But for the time being it's just fine to use the x86 version (the reported slow-down compared to native code is around 30% which is no problem for an editor).
